cmake/std/atdm/ats2/environment.sh

```diff
@@ -171,10 +171,10 @@ if [[ "$ATDM_CONFIG_COMPILER" == "CUDA-10.1.243_"* ]]; then
   # CUDA Settings
   export CUDA_LAUNCH_BLOCKING=1
   export CUDA_MANAGED_FORCE_DEVICE_ALLOC=1
-  export TMPDIR=/tmp/$(whoami)
-  if [[ ! -d ${TMPDIR} ]] ; then
-    echo "Creating ${TMPDIR} for nvcc wrapper!"
-    mkdir ${TMPDIR}
+  export TRILINOS_NVCC_WRAPPER_TMPDIR=/tmp/$(whoami)
+  if [[ ! -d ${TRILINOS_NVCC_WRAPPER_TMPDIR} ]] ; then
+    echo "Creating ${TRILINOS_NVCC_WRAPPER_TMPDIR} for nvcc wrapper!"
+    mkdir ${TRILINOS_NVCC_WRAPPER_TMPDIR}
```

Comment:
@jjellio and @e10harvey, the only reason this code was added was to get around nvcc_wrapper build errors when that directory did not exist on a given node.  Is there a reason why we should even be creating this env var `TRILINOS_NVCC_WRAPPER_TMPDIR`?  What is it used for?  If it is used for nothing, we should just delete it.

Reply:
`TRILINOS_NVCC_WRAPPER_TMPDIR` does not need to exported.

Reply:
@jjellio and @e10harvey, then can this PR just delete the `TRILINOS_NVCC_WRAPPER_TMPDIR` and `TMPDIR` vars being set set in this script all together?